test/widgets/store_test.dart

```diff
@@ -149,13 +120,18 @@ void main() {
     await tester.pumpWidget(
       Directionality(
         textDirection: TextDirection.ltr,
-        child: GlobalStoreWidget(
-          // no PerAccountStoreWidget
-          child: Builder(
-            builder: (context) {
-              final store = PerAccountStoreWidget.of(context);
-              return Text('found store, account: ${store.accountId}');
-            }))));
+        child: DeferredBuilderWidget(
+          future: testBinding.getGlobalStoreUniquely(),
+          builder: (context, store) {
+            return GlobalStoreWidget(
+              store: store,
```

Comment:
For these tests that aren't about GlobalStoreWidget and how it would formerly load data, I think they don't need DeferredBuilderWidget or an analogue; they can just use (the new) GlobalStoreWidget directly, and have the caller do its own await to get the actual global store to pass in.